benchmarks/ALIA/others/find.sl

```diff
@@ -0,0 +1,30 @@
+(set-logic ALIA)
```

Comment:
This looks suspiciously similar to the `find.sl` you added to HOLA directory. You should just remove that file and make the fix (adding an aux variable, and the missing `(< i n)` clause in `trans_fun`)

Reply:
For this pr, I removed find. I will fix add this benchmark later